tests/src/main/java/com/hazelcast/stabilizer/tests/map/MapHeapHogTest.java

```diff
@@ -1,116 +1,149 @@
 package com.hazelcast.stabilizer.tests.map;
 
-
 import com.hazelcast.core.HazelcastInstance;
 import com.hazelcast.core.IMap;
 import com.hazelcast.logging.ILogger;
 import com.hazelcast.logging.Logger;
 import com.hazelcast.stabilizer.test.TestContext;
-import com.hazelcast.stabilizer.test.annotations.Run;
+import com.hazelcast.stabilizer.test.annotations.RunWithWorker;
 import com.hazelcast.stabilizer.test.annotations.Setup;
 import com.hazelcast.stabilizer.test.annotations.Verify;
 import com.hazelcast.stabilizer.test.annotations.Warmup;
-import com.hazelcast.stabilizer.test.utils.ThreadSpawner;
+import com.hazelcast.stabilizer.worker.tasks.AbstractMonotonicWorkerTask;
+import com.hazelcast.stabilizer.worker.tasks.AbstractWorkerTask;
 
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
 
 import static com.hazelcast.stabilizer.test.utils.TestUtils.humanReadableByteCount;
 import static com.hazelcast.stabilizer.tests.helpers.HazelcastTestUtils.isMemberNode;
 import static com.hazelcast.stabilizer.tests.helpers.HazelcastTestUtils.nextKeyOwnedBy;
+import static com.hazelcast.stabilizer.utils.CommonUtils.sleepSeconds;
+import static java.lang.String.format;
 
 public class MapHeapHogTest {
 
-    private final static ILogger log = Logger.getLogger(MapHeapHogTest.class);
+    private static final int MIN_CLUSTER_SIZE = 3;
+    private static final long APPROX_ENTRY_BYTES_SIZE = 238;
+    private static final ILogger LOGGER = Logger.getLogger(MapHeapHogTest.class);
 
     // properties
-    public String basename = this.getClass().getName();
+    public String basename = this.getClass().getSimpleName();
     public int threadCount = 3;
     public int ttlHours = 24;
     public double approxHeapUsageFactor = 0.9;
+    public int logFrequency = 500000;
+    /**
+     * Configures the worker threads to use the same keys, which slows down the fill rate while keeping the load high.
+     */
+    public boolean useSameKeysOnAllWorker = true;
```

Comment:
I don't understand how this works 

the logic of the old test was to specify a max number of keys we wanted, in cluster.    agnostic to cluster size.

each member of the cluster would then loop through the MAX number of keys  putting only keys that was owned by the  member.   

this way we can control the number of keys in the map precisely, and fill up the map in the quickest possible time,  and all puts are local (only needing network for  backup's),  

I know the loop in this test is controlled by the heap space remaining,  but the logic is the same.  so how do you think, we are making this process faster, as per the title of the pr ?